components/FirebaseAuth.js

```diff
@@ -0,0 +1,51 @@
+/* globals window */
```

Comment:
FirebaseAuth.js handles our authentication with firebase.  This is where we are redirected to Googles Sign In popup